airflow/hooks/presto_hook.py

```diff
@@ -20,11 +20,13 @@ class PrestoHook(BaseHook):
     >>> ph.get_records(sql)
```

Comment:
This test somehow fails during unit testing.
`ph = PrestoHook(presto_conn_id='presto_default')` works though.